CMakeLists.txt

```diff
@@ -633,6 +635,18 @@ endif()
 install(FILES falco.yaml
 	DESTINATION "${FALCO_ETC_DIR}")
 
+if(FALCO_Coverage)
+    if (NOT (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")))
+        message(FATAL_ERROR "FALCO_Coverage requires GCC or Clang.")
```

Comment:
```suggestion
        message(FATAL_ERROR "FALCO_COVERAGE requires GCC or Clang.")
```

Reply:
I do not have access to your fork @goller. Would you pls also fix the message here with the uppercase please? :)